tests/suites/test_suite_cipher.null.data

```diff
@@ -78,15 +78,15 @@ NULL Encrypt and decrypt 16 bytes in multiple parts 4
 depends_on:MBEDTLS_CIPHER_NULL_CIPHER
 enc_dec_buf_multipart:MBEDTLS_CIPHER_NULL:0:15:1:-1:15:1:15:1
 
-NULL Encrypt and decrypt 22 bytes in multiple parts 1
+NULL Encrypt and decrypt 22 bytes in multiple parts 1 [#1]
```

Comment:
Please remove the previous numeration of tests

Reply:
Rejected as out of scope.